pyproject.toml

```diff
@@ -1,54 +1,197 @@
+package = "rasa"
+package_dir = "rasa"
+filename = "CHANGELOG.rst"
+directory = "./changelog"
+underlines = "^-_"
+title_format = "[{version}] - {project_date}"
+template = "./changelog/_template.jinja2"
+[[tool.towncrier.type]]
+directory = "removal"
+name = "Deprecations and Removals"
+showcontent = true
+
+[[tool.towncrier.type]]
+directory = "feature"
+name = "Features"
+showcontent = true
+
+[[tool.towncrier.type]]
+directory = "improvement"
+name = "Improvements"
+showcontent = true
+
+[[tool.towncrier.type]]
+directory = "bugfix"
+name = "Bugfixes"
+showcontent = true
+
+[[tool.towncrier.type]]
+directory = "doc"
+name = "Improved Documentation"
+showcontent = true
+
+[[tool.towncrier.type]]
+directory = "misc"
+name = "Miscellaneous internal changes"
+showcontent = false
+
+[tool.poetry.dependencies]
+python = "^3.6"
+boto3 = "^1.12"
+requests = "^2.23"
+matplotlib = "~3.1"
+attrs = "~19.3"
+jsonpickle = "~1.3"
+redis = "^3.4"
+numpy = "^1.16"
+scipy = "^1.4.1"
+absl-py = "~0.9"
+tensor2tensor = "^1.15"
+apscheduler = "~3.6"
+tqdm = "==4.31.0"
+networkx = "==2.4.0"
+fbmessenger = "==6.0.0"
+pykwalify = "==1.7.0"
+coloredlogs = "^10.0"
+"ruamel.yaml" = "~0.15"
+scikit-learn = "^0.22"
+slackclient = "~1.3"
+python-telegram-bot = "^11.1"
+twilio = "~6.26"
+webexteamssdk = "~1.1.1"
+mattermostwrapper = "~2.2"
+rocketchat_API = "==0.6.31"
+colorhash = "~1.0.2"
+pika = "~1.0.1"
+jsonschema = "~3.2"
+packaging = "~19.0"
+gevent = "~1.4.0"
+pytz = "^2019.1"
+rasa-sdk = "^1.7.0"
+colorclass = "~2.2"
+terminaltables = "~3.1.0"
+sanic = "~19.9.0"
+sanic-cors = "==0.9.9.post1"
+sanic-jwt = "==1.3.2"
+sanic-plugins-framework = "==0.8.2"
+cloudpickle = "~1.2.0"
+multidict = "^4.6"
+aiohttp = "~3.6"
+questionary = "==1.1.1"
+prompt-toolkit = "^2.0"
+python-socketio = "~4.4"
+python-engineio = "~3.11"
+pydot = "~1.4"
+async_generator = "~1.10"
+SQLAlchemy = "~=1.3.3"
+kafka-python = "^1.4"
+sklearn-crfsuite = "~0.3"
+psycopg2-binary = "~2.8.2"
+PyJWT = "~1.7"
+python-dateutil = "~2.8"
+tensorflow = "~2.1"
+tensorflow_hub = "~0.7"
+tensorflow-addons = "~0.7"
+tensorflow-probability = "~0.7"
+
+[tool.poetry.dev-dependencies]
+pytest-cov = "^2.8.1"
+pytest-localserver = "^0.5.0"
+pytest-sanic = "^1.1.2"
+pytest-asyncio = "^0.10.0"
+pytest-xdist = "^1.31.0"
+pytest = "^5.3.4"
+freezegun = "^0.3.14"
+responses = "^0.10.9"
+nbsphinx = "~0.5"
+aioresponses = "^0.6.2"
+moto = "==1.3.8"
+fakeredis = "^1.1.0"
+mongomock = "^3.18.0"
+black = "^19.10b0"
+flake8 = "^3.7.9"
+pytype = "^2020.1.24"
+google-cloud-storage = "^1.25.0"
+azure-storage-blob = "^12.1.0"
+coveralls = "^1.10.0"
+towncrier = "^19.2.0"
+toml = "^0.10.0"
+semantic_version = "^2.8.4"
+sphinx = "==1.8.2"
+sphinx-autobuild = "==0.7.1"
+sphinxcontrib-programoutput = "==0.11"
+pygments = "==2.2.0"
+sphinxcontrib-httpdomain = "==1.6.1"
+sphinxcontrib-websupport = "==1.1.0"
+sphinxcontrib-trio = "==1.0.2"
+sphinx-tabs = "==1.1.11"
+sphinx-autodoc-typehints = "==1.6.0"
+
+[tool.poetry.extras]
```

Comment:
is there a reason we don't have an extra for `mitie`, `kafka` and `sql` anymore?

Reply:
For some reason, `mitie` doesn't work if I specify it as an extra dependency (seems to be a poetry issue, will ask them later today), so it needs to be installed with `make install-mitie`. I hope it's a temporary measure.

I added an extra for `kafka` in the latest commit, but adding `sql` makes no sense because all of its dependencies are already installed according to `poetry show --tree`